<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MapCloud Tiler - 地图瓦片下载器</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="configGroupBox">
      <property name="title">
       <string>配置</string>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="labelConfigFile">
         <property name="text">
          <string>配置文件:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="lineEditConfigFile">
           <property name="text">
            <string>conf.toml</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnBrowseConfig">
           <property name="text">
            <string>选择配置...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelProjectName">
         <property name="text">
          <string>项目名称:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEditProjectName">
         <property name="placeholderText">
          <string>my-map</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelUrlPreset">
         <property name="text">
          <string>预设URL:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="comboBoxUrlPreset">
         <item>
          <property name="text">
           <string>自定义</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Google地图</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Google卫星</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>天地图矢量</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>天地图卫星</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labelUrl">
         <property name="text">
          <string>瓦片URL:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lineEditUrl">
         <property name="placeholderText">
          <string>http://mt0.google.com/vt/lyrs=s&amp;x={x}&amp;y={y}&amp;z={z}</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="labelGeoJson">
         <property name="text">
          <string>GeoJSON:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLineEdit" name="lineEditGeoJson">
           <property name="placeholderText">
            <string>可选：限制下载范围</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnBrowseGeoJson">
           <property name="text">
            <string>浏览...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="labelFormat">
         <property name="text">
          <string>输出格式:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="comboBoxFormat">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <item>
          <property name="text">
           <string>mbtiles</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>file</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="labelZoom">
         <property name="text">
          <string>缩放级别:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="labelMinZoom">
           <property name="text">
            <string>最小:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBoxMinZoom">
           <property name="maximum">
            <number>20</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelMaxZoom">
           <property name="text">
            <string>最大:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBoxMaxZoom">
           <property name="maximum">
            <number>20</number>
           </property>
           <property name="value">
            <number>18</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="labelWorkers">
         <property name="text">
          <string>并发数:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QSpinBox" name="spinBoxWorkers">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>50</number>
         </property>
         <property name="value">
          <number>4</number>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="labelDelay">
         <property name="text">
          <string>延时(ms):</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QSpinBox" name="spinBoxDelay">
         <property name="maximum">
          <number>10000</number>
         </property>
         <property name="singleStep">
          <number>10</number>
         </property>
         <property name="value">
          <number>30</number>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QCheckBox" name="checkBoxSkipExisting">
         <property name="text">
          <string>跳过已下载瓦片</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QCheckBox" name="checkBoxResume">
         <property name="text">
          <string>启用断点续传</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="progressGroupBox">
      <property name="title">
       <string>下载进度</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelProgress">
         <property name="text">
          <string>就绪</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="logGroupBox">
      <property name="title">
       <string>日志</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTextEdit" name="textEditLog">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnStart">
        <property name="text">
         <string>开始下载</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnPause">
        <property name="text">
         <string>暂停</string>
        </property>
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnStop">
        <property name="text">
         <string>停止</string>
        </property>
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
